PRS10001-11000/PRS10353Yeshaq.xml

```diff
@@ -45,13 +46,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <body>
          <listPerson>
             <person sex="1">
-               <persName>
-                  
-                  Yǝsḥaq
-               </persName>
-               <occupation> wr. </occupation>
```

Comment:
since this record has the specification "writer" I assume it goes up to a different Yeshaq in the index of EAe, what is your decision to use 10353 for bahr nagash is based on? We already have `PRS10343Yeshaq `for bahr nagash azmach Yeshaq, I guess you wanted to edit PRS10343Yeshaq and use PRS10343Yeshaq in the manuscript record.